build.gradle.kts

```diff
@@ -15,7 +15,7 @@ repositories {
 
 dependencies {
     //discord
-    implementation("de.chojo", "cjda-util", "2.7.3+alpha.20") {
+    implementation("de.chojo", "cjda-util", "2.7.4+alpha.20-SNAPSHOT") {
```

Comment:
I think  we want to switch to a stable version again before merging (or at least before releasing the next update)

Reply:
The review of it is assigned to you as well :P

Reply:
Oh you reviewed it already. Its merged and renovate should open a PR soon.